src/apps/loan-list/materials/utils/checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface CheckBoxProps {
+  id: string;
+  label: string;
+  hideLabel?: boolean;
+  additionalClasses?: string;
+}
+
+const CheckBox: React.FC<CheckBoxProps> = ({
+  id,
+  label,
+  hideLabel,
+  additionalClasses
+}) => {
+  return (
+    // todo add space between label adn checkbox
+    <div className={`checkbox ${additionalClasses}`}>
+      <label className="checkbox__label" htmlFor={id}>
+        <span className="checkbox__icon">
+          <svg width="20px" height="20px">
```

Comment:
Could we move this svg into a separate svg file?

Reply:
Yes. In the design system?

Reply:
Either that or like this:
[src/components/icon-favourite/icon-favourite.tsx](https://github.com/reload/dpl-react/blob/dpl-react-materiale-udgave-kort_description-list_DDFSOEG-169/src/components/icon-favourite/icon-favourite.tsx)